src/session.rs

```diff
@@ -3062,6 +3066,21 @@ impl Session {
                     self.active_view_mut().touch();
                 }
             }
+            Command::PaintColor(rgba, x, y) => {
+                self.active_view_mut().paint_color(rgba, x, y);
+            }
+            Command::PaintForeground(x, y) => {
+                let fg = self.fg;
+                self.active_view_mut().paint_color(fg, x, y);
+            }
+            Command::PaintBackground(x, y) => {
+                let bg = self.bg;
+                self.active_view_mut().paint_color(bg, x, y);
+            }
+            Command::PaintPalette(i, x, y) => {
+                let c = self.palette.colors[i as usize];
```

Comment:
We're going to want to use `get` here, otherwise this will crash the program if `i` is not in `colors`

Reply:
I'm sorry. I'm quite new to Rust so I didn't know the best way to prevent it from going out of bounds.

Reply:
Updated to use `.get()`